explainer.md

```diff
@@ -199,26 +207,34 @@ Creating a Realm that runs in a separate process is another alternative, while a
 
 This alternative was discarded for two main reasons:
 
-1. There are existing mechanism to achieve this today in both browsers, and nodejs. E.g.: cross domain iframes, workers, etc. They seem to be good enough when asynchronous communication is sufficient to implement the feature.
-2. Asynchronous communication is a deal-breaker for many use-cases, specially when security is __not__ an issue, and sometimes it just added complexity for cases where a same-process Realm is sufficient.
+1. There are existing mechanism to achieve this today in both browsers, and nodejs. E.g.: cross-origin iframes, workers, etc. They seem to be good enough when asynchronous communication is sufficient to implement the feature.
+2. Asynchronous communication is a deal-breaker for many use-cases, specifically when security is __not__ an issue, and sometimes it just added complexity for cases where a same-process Realm is sufficient.
 
-E.g. Google AMP run in a cross domain iframe, and just want more control about what code they executed in that cross domain application.
+E.g. Google AMP run in a cross-origin iframe, and just want more control about what code they executed in that cross-origin application.
 
 There are some identified challenges explained within the current use cases for ShadowRealms such as the [WorkerDOM Virtualization challenge for Google AMP](#DOMVirtualization) and the current use of [JSDOM and Node VM modules](#JSDOMvmModules) that would be better placed using an interoperable ShadowRealm API as presented by this proposal.
 
+## <a name='Security'></a>Security
+
+__ShadowRealms, by design, cannot securely run *untrusted* code.__ While claims to the contrary have been made in the past, we *strongly discourage* using ShadowRealms for this purpose.
+
+While direct cross-realm object access is disabled, ShadowRealms by design share a heap and thus a process. Attackers can read cross-realm memory using timing attacks (e.g. [Spectre](https://leaky.page/)) without relying on any exploits in the JavaScript engine. Attackers leveraging exploits can go further and freely write cross-realm memory. The design goals of ShadowRealms make it fundamentally impossible for JavaScript engines to improve on this situation.
```

Comment:
How about: "share the process, memory heap, and event loop with the origin (application) which created the ShadowRealm" to make it clear that ShadowRealms don't just share it with other ShadowRealms but with the application itself?

(I'm not sure about the event loop FWIW, this is just a guess.)